.github/workflows/build-deps.yml

```diff
@@ -0,0 +1,196 @@
+name: Build emsdk dependencies
+
+on:
+  workflow_dispatch:
+    inputs:
+      llvm_build:
+        description: 'LLVM Webassembly arm64'
+        required: false
+        default: false
+        type: boolean
+      llvm_build_cross_compilation:
+        description: 'LLVM Webassembly arm64 (cross compilation)'
+        required: false
+        default: false
+        type: boolean
+      emscripten_build:
+        description: 'Emscripten'
+        required: false
+        default: false
+        type: boolean
+      nodejs_build:
+        description: 'Node.js arm64'
+        required: false
+        default: false
+        type: boolean
+      emsdk_build:
+        description: 'emsdk arm64'
+        required: false
+        default: false
+        type: boolean
+
+jobs:
+  build-llvm-webassembly-arm64:
```

Comment:
Do we want to keep `both build-llvm-webassembly-arm64` and `build-llvm-webassembly-arm64_cross_compilation` jobs? What are the use cases of both in particular? Can they be merged and whether it will be a cross-compilation controlled by a parameter?

Reply:
cross compilation is needed for default github runner. It is not finished yet. I would keep it for now as it is. However it might be fixed already by changing build process for main llvm job. It should be verified.